package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
```

Comment:
bell, sadly, doesn't have a changelog. changes from 2.0.0 to 2.1.0:

- add google example
- add forceHttps option
- remove `pack` from readme for hapi 8